homeassistant/components/systemair/sensor.py

```diff
@@ -0,0 +1,505 @@
+            'Alarm Supply Temperature Sensor',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_SUPPLY_TEMP_SENSOR
+        ],
+        ATTR_ALARM_INDOOR_TEMP_SENSOR: [
+            'Alarm Indoor Temperature Sensor',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_INDOOR_TEMP_SENSOR
+        ],
+        ATTR_ALARM_EXTRACT_TEMP_SENSOR: [
+            'Alarm Extract Temperature Sensor',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_EXTRACT_TEMP_SENSOR
+        ],
+        ATTR_ALARM_PREHEATER_TEMP_SENSOR: [
+            'Alarm Preheater Temperature Sensor',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_PREHEATER_TEMP_SENSOR
+        ],
+        ATTR_ALARM_EFFICIENCY_TEMP_SENSOR: [
+            'Alarm Efficiency Temperature Sensor',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_EFFICIENCY_TEMP_SENSOR
+        ],
+        ATTR_ALARM_PDM_RH: [
+            'Alarm PDM Relative Humidity',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_PDM_RH
+        ],
+        ATTR_ALARM_PDM_TEMP: [
+            'Alarm PDM Temperature Sensor',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_PDM_TEMP
+        ],
+        ATTR_ALARM_CHANGE_FILTER: [
+            'Alarm Filter Replacement',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_CHANGE_FILTER
+        ],
+        ATTR_ALARM_EXTRA_CONTROLLER: [
+            'Alarm Extra Controller Malfunction',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_EXTRA_CONTROLLER
+        ],
+        ATTR_ALARM_EXTERNAL_STOP: [
+            'Alarm External Stop',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_EXTERNAL_STOP
+        ],
+
+        ATTR_ALARM_MANUAL_STOP: [
+            'Alarm Stopped Manually',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_MANUAL_STOP
+        ],
+
+        ATTR_ALARM_REHEATER_OVERHEAT: [
+            'Alarm Overheated Reheater',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_REHEATER_OVERHEAT
+        ],
+
+        ATTR_ALARM_CO2: [
+            'Alarm Co2 Quality',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_CO2
+        ],
+
+        ATTR_ALARM_RH: [
+            'Alarm Humidity Quality',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_RH
+        ],
+
+        ATTR_ALARM_INCORRECT_MANUAL_MODE: [
+            'Alarm Incorrect Manual Mode',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_INCORRECT_MANUAL_MODE
+        ],
+
+        ATTR_TARGET_TEMPERATURE: [
+            'Target Temperature',
+            TEMP_CELSIUS,
+            'mdi:alert',
+            SENSOR_TARGET_TEMPERATURE
+        ]
+
+    }
+
+    sab = hass.data[DOMAIN]
+
+    sensors = []
+    new_sensors = {
+        ATTR_EXTRACT_TEMPERATURE
+    }
+
+    for resource in config[CONF_RESOURCES]:
```

Comment:
We don't allow this type of config option. See:
https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md